puppet/zulip/files/logrotate/zulip

```diff
@@ -1,8 +1,13 @@
-/var/log/zulip/server.log /var/log/zulip/workers.log /var/log/zulip/manage.log {
+/var/log/zulip/*.log {
 	missingok
 	rotate 10
 	size 1G
 	compress
 	delaycompress
 	notifempty
+        create 644 zulip zulip
+        sharedscripts
+        postrotate
+                supervisorctl signal SIGHUP all >/dev/null 2>&1
+        endscript
```

Comment:
SIGHUP is the standard signal used during regular logrotation processes to release file handles. If that causes a full server restart on some of the Zulip processes, and you see there's a better/safer way to achieve the same result (file handles release), definitely go for it :)